packages/appstore/src/components/wallet-header/wallet-currency-card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import type { TAccountCategory, TWalletSvgCurrency } from 'Types';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import { isCryptocurrency } from '@deriv/shared';
+
+type TWalletCurrencyCard = {
+    account_type: TAccountCategory;
+    currency: TWalletSvgCurrency;
+};
+
+const WalletCurrencyCard = observer(({ account_type, currency }: TWalletCurrencyCard) => {
+    const {
+        ui: { is_dark_mode_on },
+    } = useStore();
+
+    const is_demo = account_type === 'demo';
+    const theme = is_dark_mode_on ? '--dark' : '';
+    const class_currency = is_demo ? 'demo' : currency.toLowerCase();
+
+    const currency_icon_path = getWalletCurrencyIcon(is_demo ? 'demo' : currency, is_dark_mode_on);
+
+    // add check (currency !== 'USDT') because response from BE doesn't have USDT currency, just UST
+    const is_fiat = !isCryptocurrency(currency) && currency !== 'USDT';
+    const currency_icon_name = currency_icon_path;
```

Comment:
why you define new variable which is equal to the other one?

Reply:
@mahdiyeh-deriv It's because I'm trying to refactor the code too fast and my hands are faster than my brain :)